app.js

```diff
@@ -24,10 +25,12 @@ app.use('/crit', critRoute);
 app.use( express.static('static') );
 app.set('view engine', 'ejs');
 
+console.log(bcrypt);
+
 // Route for Timeline
 
 app.get('/', (req, res) => {
-	console.log(req.session.loggedin)
+	console.log(req.session.loggedin);
```

Comment:
We can probably remove the console.log debugging stuff now that you have things working. On a real webserver when we deploy this, all of these console.log() messages will end up cluttering up our log files.